zfit_physics/models/pdf_cruijff.py

```diff
@@ -0,0 +1,100 @@
+from typing import Optional
+
+import zfit
+from zfit import z
+from zfit.util import ztyping
+from zfit.z import numpy as znp
+
+
+@z.function(wraps="tensor")
+def cruijff_pdf_func(x, mu, sigmal, alphal, sigmar, alphar):
+    """Calculate the Cruijff PDF.
+
+    Args:
+        x: value(s) for which the PDF will be calculated.
+        mu: Mean value
+        sigmal: Left width parameter.
+        alphal: Left tail acceleration parameter.
+        sigmar: Right width parameter.
+        alphar: Right tail acceleration parameter.
+
+    Returns:
+        `tf.Tensor`: The calculated PDF values.
+
+    Notes:
+        Implementation from https://arxiv.org/abs/1005.4087, citation 22, and http://hdl.handle.net/1794/9022, Equation IV.3
+    """
+    cond = znp.less(x, mu)
+
+    func = znp.where(
+        cond,
+        znp.exp(-0.5 * znp.square((x - mu) / sigmal) / (1 + alphal * znp.square((x - mu) / sigmal))),
+        znp.exp(-0.5 * znp.square((x - mu) / sigmar) / (1 + alphar * znp.square((x - mu) / sigmar))),
+    )
+    return func
```

Comment:
```suggestion
# compute only once (in graph this _may_ be optimized anyways, but surely not in eager)
xminmu = (x - mu)
tleft = znp.square(xminmu / sigmal)
tright = znp.square(xminmu / sigmar)
exponent = znp.where(
    cond,
    tleft / (1 + alphal * tleft),
    tright / (1 + alphar * tright),
)
value = znp.exp(-0.5 * exponent)
    return value
```
(also, I renamed func-> value, also `out` or anything is fine, but `func` is often used as a callable, absolutely minor detail)